roles/davinci_resolve/defaults/main.yml

```diff
@@ -17,3 +17,4 @@ davinci_resolve:
   city: Dover
   street: 838 Walker Road
   user_agent: "Mozilla/5.0 (X11; Linux {{ ansible_architecture }}; rv:131.0) Gecko/20100101 Firefox/131.0"
+# davinci_resolve_zip_file: '/path/to/davinciresoolve.zip' # <- enable it explicitly in `ansible-playbook` 
```

Comment:
What's the purpose of this commented code?

Reply:
To show this is a variable that can be set during the execution of the role, with an example on what it should contain. Should I remove the line? :thinking:

Reply:
yeah I'm not sure... did you check if plays well with https://github.com/leinardi/JDInstaller/blob/master/generate-group-vars.sh?